typescript/widgets/src/logos/protocols.ts

```diff
@@ -16,4 +16,5 @@ export const PROTOCOL_TO_LOGO: Record<
   [ProtocolType.Cosmos]: CosmosLogo,
   [ProtocolType.CosmosNative]: CosmosLogo,
   [ProtocolType.Starknet]: StarknetLogo,
+  [ProtocolType.Radix]: EthereumLogo,
```

Comment:
imo may as well add the RadixLogo here now, or add a // TODO to change it so we don't forget